### DISCUSSION TITLE
GitHub Copilot  Extensions does not appear in VisualStudio 2019  Market place.

### DISCUSSION THREAD

**Discussion by kirantoday** [community/community#44241] [Copilot Conversations]
### Select Topic Area

Product Feedback

### Body

[](url)I have VisualStudio 2019 and i opened the Extensiosn->Visual Studio MarketPlace and search for GitHub Copilot .  The Extensions does not show up. 
Is there any fix for this?

**Comment by tilmann1** [comment]
@kirantoday The Copilot VS extension is not supported with VS 2019. Using the latest stable version of VS 2022 is recommended to get the best Copilot experience, currently that's VS 17.4.4 which was released earlier this week. Thanks!

**Reply by KEYLEDGER** [reply]
Did I dream that or was VS2019 supported at some point? Thanks

**Reply by tilmann1** [reply]
@Crypties No, I can confirm that VS 2019 has never been supported by the Copilot VS extension.

**Reply by KEYLEDGER** [reply]
Okay. A CRC check is in order on my memory I guess. By any chance do you know where I could find stats on current market shares of VS 2019 vs 2022, Vs2022 adoption rate in enterprises etc? Had no luck looking for this so far.
